README.md

```diff
@@ -1 +1,2 @@
-placeholder, please ignore.
+# gpii-express
+A fluid module to load express.js
```

Comment:
Sorry, that was boilerplate that I obviously missed updating.  This has now been greatly expanded.